plugins/dataframe-gradle-plugin/src/main/kotlin/org/jetbrains/dataframe/gradle/toCamelCaseByDelimiters.kt

```diff
@@ -1,9 +1,11 @@
 package org.jetbrains.dataframe.gradle
 
+import java.util.Locale
+
 fun String.toCamelCaseByDelimiters(delimiters: Regex): String {
-    return split(delimiters).joinToCamelCaseString().decapitalize()
+    return split(delimiters).joinToCamelCaseString().replaceFirstChar { it.lowercase(Locale.getDefault()) }
 }
 
 fun List<String>.joinToCamelCaseString(): String {
-    return joinToString(separator = "") { it.capitalize() }
+    return joinToString(separator = "") { s -> s.replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() } }
```

Comment:
Sorry, changes in the behaviour is not clear for me here. Is it related to the KSP update, isn't it?

Reply:
`capitalize` was deprecated some time ago. I come across the warning and decided to fix it